cEP-0029.md

```diff
@@ -0,0 +1,342 @@
+              ('max_line_length', <Setting object(key='max_line_length', value='80', origin='/Users/prajwalm/Documents/coala/coala/.coafile', from_cli=False, to_append=False) at 0x1092cda20>),
+              ('use_spaces', <Setting object(key='use_spaces', value='True', origin='/Users/prajwalm/Documents/coala/coala/.coafile', from_cli=False, to_append=False) at 0x1092cd898>)]), defaults=None,
+              language=None, name='Default') at 0x108bdf550>)])
+```
+
+into
+
+```python
+OrderedDict([('Default',
+              OrderedDict([('files',
+                            ['*.py',
+                             'coalib/**/*.py',
+                             './coala',
+                             'tests/**/*.py',
+                             'docs/conf.py']),
+                           ('ignore',
+                            ['tests/bearlib/languages/documentation/documentation_extraction_testdata/*.py',
+                             'tests/collecting/collectors_test_dir/bears/incorrect_bear.py',
+                             'coalib/misc/Asyncio.py']),
+                           ('max_line_length', 80),
+                           ('use_spaces', True)]))])
+```
+into
+
+
+### Tool to convert .coafile to TOML config file
+
+A new flag will be created to allow .coafile to
+be converted to TOML config files and vice versa.
+
+```bash
+$ coala --create-toml .coafile
+$ coala --create-coafile .coafile.toml
+```
+
+The loaded sections from either .coafile or .coafile.toml
+can be supplied to:
+- TomlConfWriter to create TOML config file
+- ConfWriter to create coafile
+
+
+### Support coala-quickstart to generate config files in TOML.
+
+coala-quickstart will be made to generate config files in TOML.
+A new option -T that will allow config files to be created in
+TOML.
+
+New functions `write_toml_config` and `write_toml_info` will
+be created which will perform the functions similar to existing
+`write_coafile` and `write_info` functions but with respect to TOML.
+
+Regarding green mode, I will create `write_toml_sections` function
+similar to `write_sections` for TOML. Also the function
+`generate_green_mode_sections` will be modified to accommodate
+for TOML config file.
+
+
+### Develop TOMLLintBear
+
+coala-bears currently has bear for TOML called `TOMLBear.py` which
+uses a external linter. But it requires Go to be installed.
+So this bear can mainly be used only by Go users.
+
+The task is to create `TOMLLintBear` in python. This way it can
+be used by all users without any additional language requirements
+as python comes installed in most computers.
+
+The bear shall catch syntax errors in a TOML file. This shall be
+done in 2 passes.
+- In the first pass, the TOML file shall be validated per character
+- In the second pass, the TOML file shall be validated per line
+
+This approach is similar to the toml library.
+
+
+### A repository of good config files to allow users to easily bootstrap their projects.
+
```

Comment:
Is this section unfinished or deliberately left empty?

Reply:
If this project is carried, should the foundation of it -- i.e. adding some example config files -- be started within this project? I feel that some examples and community-provided configurations sould be present from the start.